.gitignore

```diff
@@ -2,33 +2,353 @@
 /build
 AssemblyInfo.cs
 
-# Standard VS.NET and ReSharper Foo
-.vs/*
-src/*/obj
-src/*/bin
-samples/*/obj
-samples/*/bin
-*.csproj.user
-*ReSharper.user
-_ReSharper*
-*.suo
 *.cache
 * Thumbs.db
```

Comment:
Thumbs.db is an example of something I'd handle on a system globally, we don't even have it in Core.